### ISSUE TITLE
v3 vor the normal 0.1

### ISSUE THREAD

**Issue by lukassmall** [zruncho3d/ZeroPanels#9]
Are there files of v3 for the stock version of the voron v0.1?

**Comment by zruncho3d** [comment]
No, but user Adooze did a nice V0-oriented port, and others have done similar ones.  I'd suggest searching on the Voron and DoomCube discords; you should find them and avoid the work.   Let me know, as I'd like to link those here.

**Comment by jamincollins** [comment]
This looks like it:
https://github.com/adooze/ZeroPanels_for_Stock_V0.2/tree/main
